.github/workflows/nix-fmt-checks.yaml

```diff
@@ -0,0 +1,27 @@
+name: Nix formatter checks
+
+on:
+  pull_request:
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v18
+
+      - name: Run nix formatter tool
+        run: nix fmt
+
+      - name: Verify Changed files
+        uses: tj-actions/verify-changed-files@v13
+        id: verify_changed_files
+
+      - name: Files has changed
+        if: steps.verify_changed_files.outputs.files_changed == 'true'
+        run: |
+          echo "Repository has uncommitted changes, run nix fmt locally and commit the result."
+          exit 1
```

Comment:
```suggestion
        run: nix fmt . -- --check
```

Reply:
Nice ! Going to take care of that tomorrow, went to bed already.

Reply:
Fixed!